src/middleware/serverMiddlewareLoader.middleware.ts

```diff
@@ -74,17 +75,22 @@ export default class ServerMiddlewareLoader {
 
     private prepareCSPConfig (nonce: string): ContentSecurityPolicyOptions {
         const piwikConfig = ServerMiddlewareLoader.extractPiwikHost(this.PIWIK_CONFIG)
+        let cdnHostPrefix: string = this.CDN_HOST
+        if ( this.CDN_HOST_LONG_PREFIX &&
```

Comment:
Do you need a new variable couldn't you just search `cdnHostPrefix` for a slash (that is it has a long prefix)?

Reply:
- the new variable `cdnHostPrefix` is required so that the setting of `ContentSecurityPolicyOptions` doesn't use `this.CDN_HOST` but uses the value `this.CDN_HOST + '/'` 
- a new ENV var (the boolean flag `CDN_HOST_LONG_PREFIX`) is added so that the ending slash `/` is appended only when the flag is on. 

A slash already somewhere in `CDN_HOST` doesn't mean we want to activate the behaviour

Here we want to append a slash `'/'` to fix only the CSP part of this service which started failing when we added a longer prefix in `CDN_HOST`
The [specs](https://www.w3.org/TR/CSP/#match-paths) for this part of CSP are n. 3:
```
Let exact match be false if the final character of path A is the U+002F SOLIDUS character (/), and true otherwise.```